hotshot-state-prover/src/circuit.rs

```diff
@@ -68,10 +68,6 @@ impl LightClientStateVar {
                 circuit.create_public_variable(view_number_f)?,
                 circuit.create_public_variable(block_height_f)?,
                 circuit.create_public_variable(state.block_comm_root)?,
-                circuit.create_public_variable(state.fee_ledger_comm)?,
-                circuit.create_public_variable(state.stake_table_comm.0)?,
-                circuit.create_public_variable(state.stake_table_comm.1)?,
-                circuit.create_public_variable(state.stake_table_comm.2)?,
```

Comment:
ah this is wrong actually. 🤦 
(i wasted some time assuming our Public input is reduced to 4) 

we should still keep the stake table commitments in the public inputs for security/integrity!!

but we can safely remove `fee_ledger_comm`, so PI.length is 7 now.

cc @mrain @alysiahuggins

Reply:
ah, I need to fix `HotShot` side as well for the struct declaration.

Reply:
fixed in [ab81b0b](https://github.com/EspressoSystems/espresso-sequencer/pull/1986/commits/ab81b0bad7497dbcedc4434cd1881cb6891d7c58)